recipes-openxt/qemu-dm/qemu-dm.inc

```diff
@@ -25,7 +25,6 @@ SRC_URI += "file://compile-time-stubdom-flag.patch \
             file://surfman-dcl.patch \
             file://audio-policy.patch \
             file://msix-cap-disable.patch \
-            file://xenstore-based-ISO-media-change-detection-for-both-s.patch \
```

Comment:
If you are dropping the patch queue, might I suggest removing the patch so there is no confusion about carrying the patch forward on the next uprev of qemu.

Reply:
Ack, fixed.